hiring/hiring-process/engineering-hiring/scoring-a-take-home-exercise.md

```diff
@@ -1,53 +1,58 @@
+| Polished UI, with thought put into the user experience and/or with nice transitions and animations.                                                                                                                              | 2     |
+| **Documentation**                                                                                                                                                                                                                |       |
+| No documentation in the repository.                                                                                                                                                                                              | 0     |
+| Some documentation with basic setup instructions or feature descriptions.                                                                                                                                                        | 1     |
+| Detailed documentation with setup, screenshots, configuration instructions, etc.                                                                                                                                                 | 2     |
+| **Commit History \(See** [**Note 1**](scoring-a-take-home-exercise.md#note-1) **below\)**                                                                                                                                        |       |
+| The repository has a few large commits \(Exceptions are commits with auto-generated code like Room schemas or the initial commit when initialising a project through an IDE or a CLI\).                                          | 0     |
+| The repository has some small-ish commits with clear \(what the commit does\) messages.                                                                                                                                          | 1     |
+| The repository has many tiny, atomic commits with clear, descriptive \(what the commit does and why, when appropriate\) messages.                                                                                                | 2     |
+| **Testing**                                                                                                                                                                                                                      |       |
+| There are either no tests, or a few tests that aren't really that useful.                                                                                                                                                        | 0     |
+| Tests for business logic are present, but they are tested via instrumentation/UI tests.                                                                                                                                          | 1     |
```

Comment:
We should also mention what should be the score if there are limited unit tests.

Reply:
I remember seeing some submissions where there were unit tests but they were not complete.

Reply:
If the tests were limited, but useful, I'd generally put those at 1 point.

Reply:
Can you mention this in the point for 1 so as to reduce subjectivity?

Reply:
Yup, I'll do this.

Reply:
Done.